src/components/panel/panel.spec.js

```diff
@@ -1538,6 +1538,52 @@ describe('$mdPanel', function() {
         expect(panelCss.right).toEqual('0px');
       });
 
+      it('start in ltr', function() {
+        var start = '50px';
+        config['position'] = mdPanelPosition.absolute().start(start);
+
+        openPanel(config);
+
+        var panelCss = document.querySelector(PANEL_EL).style;
+        expect(panelCss.left).toEqual(start);
+      });
+
+      it('start in rtl', function() {
+        var start = '50px';
+        mdPanelPosition._isRTL = true;
+        config['position'] = mdPanelPosition.absolute().start(start);
+
+        openPanel(config);
+
+        var panelCss = document.querySelector(PANEL_EL).style;
+        expect(panelCss.right).toEqual(start);
+
+        mdPanelPosition._isRTL = false;
```

Comment:
Does it make sense to add this to an afterEach to be extra safe that it's reset?